src/commands/get_pseudo_random.rs

```diff
@@ -0,0 +1,36 @@
+//! Get Pseudo Random Bytes
+//!
+//! <https://developers.yubico.com/YubiHSM2/Commands/Get_Pseudo_Random.html>
+//!
+use super::{Command, Response};
+use {CommandType, Connector, Session, SessionError};
+
+/// Get some number of bytes of pseudo random data generated on the device
+pub fn get_pseudo_random<C: Connector>(
+    session: &mut Session<C>,
+    bytes: u16,
+) -> Result<GetPseudoRandomResponse, SessionError> {
+    session.send_encrypted_command(GetPseudoRandomCommand { bytes })
+}
+
+/// Request parameters for `commands::get_pseudo_random`
+#[derive(Serialize, Deserialize, Debug)]
+pub(crate) struct GetPseudoRandomCommand {
+    /// Number of random bytes to return
+    pub bytes: u16,
+}
+
+impl Command for GetPseudoRandomCommand {
+    type ResponseType = GetPseudoRandomResponse;
+}
+
+/// Response from `commands::get_pseudo_random`
+#[derive(Serialize, Deserialize, Debug)]
+pub struct GetPseudoRandomResponse {
```

Comment:
If you have `get_pseudo_random` return a `Vec<u8>` you can mark this as `pub(crate)`